components/tools/OmeroWeb/test/integration/test_api_containers.py

```diff
@@ -156,6 +171,22 @@ def screen_plates(self, user1):
 
         screen = get_update_service(user1).saveAndReturnObject(screen)
         plate = get_update_service(user1).saveAndReturnObject(plate)
+
+        # Add well to first plate
```

Comment:
I've added PlateAcquisition testing to ```test_api_wells.py``` but not duplicated this in ```test_api_containers.py``` because this test is really concerned with testing urls: Screen -> Plate -> Wells and we don't need to cover PlateAcquisitions twice. Also, it's nice to test that we can load Wells with NO Plate Acquisitions which we're not testing anywhere else now.